sonaranalyzer-dotnet/its/expected/Ember-MM/Ember.Plugins-{9496C697-5AFD-4813-AEDC-AF33FACEADF0}-S4056.json

```diff
@@ -19,19 +19,6 @@
 "location":  {
 "uri":  "sources\Ember-MM\Ember.Plugins\Ember\EmberWorkAroundPlugin.cs",
 "region":  {
-"startLine":  63,
```

Comment:
`EndsWith(String value);` vs `EndsWith(String value, bool ignoreCase, CultureInfo culture);` - not direct overload

Reply:
answer: we have a rule for StringComparison https://rules.sonarsource.com/csharp/RSPEC-4058